src/service/fixed-layer.js

```diff
@@ -449,23 +456,23 @@ export class FixedLayer {
         // 1. Unset top from previous mutates and set bottom to an extremely
         // large value (to catch cases where sticky-tops are in a long way
         // down inside a scroller).
-        for (let i = 0; i < elements.length; i++) {
-          setImportantStyles(elements[i].element, {
+        elements.forEach(fe => {
```

Comment:
bytes-saving idea: move the bodies of these three `forEach` loops into one `forEach`. That should get rid of some extra boilerplate in the final build.

Reply:
Not sure if that affects the mutate-read cycle?

Reply:
I don't understand why it would since `Array.prototype.forEach` is synchronous.

Reply:
It was paranoia on my part, but I just realized why it's done this way.

Step 1 is `mutate all`.
Step 2 is `read all`.
Step 3 is `mutate all again`

Vsync contract is already broken here, so if we were to put all in the same loop, we'd be causing reflows for each element, instead of one reflow per loop.

Reply:
Ah it was right there in the comment for me too : P Thanks for taking the time to explain